tests/beaker_tests/cisco_overlay_global/test_overlay_global.rb

```diff
@@ -103,8 +102,12 @@
 
 tests['preclean'] = {
   manifest_props: '',
-  code:           [0, 2],
-  resource_props: {},
+  code:           [0],
+  resource_props: {
+    # These properties always exist
+    'dup_host_mac_detection_host_moves' => '5',
+    'dup_host_mac_detection_timeout'    => '180',
+  },
 }
 
 tests['default_properties'] = {
```

Comment:
Please add `platform` key `n9k` to the top-level test hash and add call to `platform_supports_test` api.

https://github.com/cisco/cisco-network-puppet-module/blob/release_1.2.0/tests/beaker_tests/cisco_vxlan_vtep/test_vxlan_vtep.rb#L75

https://github.com/cisco/cisco-network-puppet-module/blob/release_1.2.0/tests/beaker_tests/cisco_vxlan_vtep/test_vxlan_vtep.rb#L196